res/css/views/auth/_AuthBody.scss

```diff
@@ -94,6 +94,22 @@ limitations under the License.
             background-color: $authpage-focus-bg-color;
         }
     }
+
+    .mx_recaptchaContainer {
+        .mx_recaptchaContainer_info,
```

Comment:
`mx_recaptchaContainer_info` is a specific selector, it does not need to be nested.

Reply:
Generally nesting should protect you from unexpected visual regressions. If you don't have a strong opinion against it, I would rather keep this nested until another element outside of it requires this.